string msgType
string processMsg
---
int32 result
